### PR TITLE
Fix error and Add module Patient Management

### PR DESCRIPTION
**Pull request by TrungLuong1194**

### CODE REVIEW

File: models/models.py [hunk 1]
```diff
@@ -12,31 +12,84 @@
...
+	# One2many
+    patient_ids = fields.One2many('patient_management.patient', 'doctor_id', string="Patient")
+
+    # Computed Patient
+    number_patient = fields.Integer(string="Number of Patient", compute='_computed_patient')
+
+    @api.depends('patient_ids')
+    def _computed_patient(self):
+        for record in self:
+            if not record.patient_ids:
```
Comment (namnh139) [inline models/models.py]: @TrungLuong1194 we don't need this conditional

File: models/models.py [hunk 2]
```diff
@@ -12,31 +12,84 @@
...
+
+    @api.depends('doctor_ids')
+    def _computed_doctor(self):
+        for record in self:
+            record.number_doctor = len(record.doctor_ids)
+
+    # Onchange
+    @api.onchange('disease_beds')
+    def _verify_valid_beds(self):
+        if (self.disease_beds <= 0):
```
Comment (namnh139) [inline models/models.py]: @TrungLuong1194 Remove `( )`

**Comment by namnh139** [comment]
LGTM :+1:

**Comment by vuonghv** [comment]
LGTM :+1:

**Pull request merged**